app/controllers/miq_policy_controller/policies.rb

```diff
@@ -225,7 +225,7 @@ def excluded_event?(event)
 
   def policy_get_all_folders(parent = nil)
     if !parent.nil?
-      @folders = %w(Host Vm)
+      @folders = %w(Host Vm ContainerImage)
```

Comment:
Does this make sense to live in MiqPolicy as a constant instead of in a controller?